### ISSUE TITLE
ZMQ jll for Apple Silicon platform needed

### ISSUE THREAD

**Issue by domschl** [JuliaInterop/ZMQ.jl#210]
In order to use jupyter, a prebuilt version of ZMQ jll for :aarch64 (apple silicon) is needed.
Patching ZMQ.jl to use a system arm64 version of zeromq install via `brew` works fine.

**Comment by giordano** [comment]
Fixed by https://github.com/JuliaPackaging/Yggdrasil/pull/3194.  I confirm `ZMQ.jl` passes tests with `ZeroMQ_jll` v4.3.4, but the IJulia kernel doesn't start for me in a Jupyter notebook.

**Comment by stevengj** [comment]
Closed as fixed upstream.   If there are other problems with IJulia on m1, that should be a separate issue.

**Comment by domschl** [comment]
Confirmed it works now, thank you.

Two comments:
* I somehow did not automatically receive the new version of `ZeroMQ_jll`, I had to deinstall `ZeroMQ_jll` and `ZMQ` and at some point a new version was installed and it worked.
* There seems to be an incompatibility with the Jupyter kernel of IJulia: removing the old kernel and rebuilding IJulia fixed this, so IJulia is now working too without any patches on M1 with julia master.
